bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+            - Adverbs that can weaken meaning: really, very, extremely, etc.
+            - Readibility of sentences
+            - Simpler expressions
+            - Weasel words
+            - If a sentence is preceded by a space
+            - If there is no space between a sentence and its ending
+              punctuation
+            - If sentences are starting with uppercase letter
+        :param check_offensive_expressions:
+            This rule helps you find gender favouring, polarising, race
+            related, religion inconsiderate, or other unequal phrasing.
+        :param check_common_misspellings:
+            This rule helps to find common misspellings from Wikipedia's
+            list of common misspellings.
+        :param write_good:
+            This rule checks your English styles with `write-good
+            <https://github.com/btford/write-good>`_.
+        :param check_relative_links:
+            This rule enables dead link checks against relative URIs.
+            Note that you also have to specify the ``base_uri`` to make this
+            option work.
+        :param base_uri:
+            The base URI to be used for resolving relative URIs.
+        :param ignore_links:
+            A list of URIs to be ignored, i.e. skipped from availability
+            checks.
+        """
+        if textlint_config:
+            return None
+        else:
+            options = {
+                'no-todo': check_todos,
+                'no-start-duplicated-conjunction':
+                    dont_start_with_duplicated_conjunction,
+                'no-empty-section': no_empty_section,
+                'date-weekday-mismatch': check_date_weekday_mismatch,
+                'ginger': check_grammar,
+                'max-number-of-lines': {
+                    'max': max_lines_per_file
+                },
+                'max-comma': {
+                    'max': max_comma_per_sentence
+                },
+                'ng-word': {
+                    'words': check_no_good_words
+                },
+                'period-in-list-item': period_in_list_item,
+                'unexpanded-acronym': {
+                    'min_acronym_len': minimum_acronym_length,
+                    'max_acronym_len': maximum_acronym_length,
+                    'ignore_acronyms': ignore_acronyms
+                },
+                'rousseau': check_rousseau,
+                'alex': check_offensive_expressions,
+                'common-misspellings': check_common_misspellings,
+                'write-good': write_good,
+                'no-dead-link': {
+                    'checkRelative': check_relative_links,
+                    'baseURI': base_uri,
+                    'ignore': ignore_links
+                }
+            }
+
+            default_config = {
+                'rules': options,
+                'plugins': ['rst', 'html', 'asciidoc-loose']
+            }
+            return json.dumps(default_config)
```

Comment:
up to you, though I would kinda like to see if there's a \n before `return`. Imo that allows to distinguish a return better :+1: